docs/src/pages/components/autocomplete/IntegrationReactSelect.tsx

```diff
@@ -311,6 +311,16 @@ function IntegrationReactSelect() {
         <Select
           classes={classes}
           styles={selectStyles}
+          TextFieldProps={{
+            label: 'Country',
+            InputProps: {
+              id: 'react-select-single',
+            },
+            InputLabelProps: {
+              htmlFor: 'react-select-single',
+              shrink: true,
```

Comment:
It's not needed.

Reply:
Either this or we remove the placeholder. Without this the placeholder is hidden.